app/helpers/container_image_helper/textual_summary.rb

```diff
@@ -67,6 +67,18 @@ def textual_product_name
       {:label => _("Product Name"), :value => name} if name
     end
 
+    def textual_author
+      {:label => _('Author'), :value => @record.author}
+    end
+
+    def textual_entrypoint
+      {:label => _('Entrypoint'), :value => @record.entrypoint}
```

Comment:
So if there's value in `@record.entrypoint` it's gonna be Array of String and it will crash with
```
F, [2018-01-10T15:27:47.840813 #70103] FATAL -- : Error caught: [ActionView::Template::Error] no implicit conversion of Symbol into Integer
/Users/zita/Desktop/ManageIQ/manageiq-ui-classic/app/helpers/application_helper.rb:156:in `[]'
/Users/zita/Desktop/ManageIQ/manageiq-ui-classic/app/helpers/application_helper.rb:156:in `block in hover_class'
/Users/zita/Desktop/ManageIQ/manageiq-ui-classic/app/helpers/application_helper.rb:156:in `any?'
/Users/zita/Desktop/ManageIQ/manageiq-ui-classic/app/helpers/application_helper.rb:156:in `hover_class'
/Users/zita/Desktop/ManageIQ/manageiq-ui-classic/app/views/shared/summary/_textual.html.haml:18:in `block in ___sers_zita__esktop__anage___manageiq_ui_classic_app_views_shared_summary__textual_html_haml__1784846946020889247_70132237067240'
/Users/zita/Desktop/ManageIQ/manageiq-ui-classic/app/views/shared/summary/_textual.html.haml:9:in `each'
/Users/zita/Desktop/ManageIQ/manageiq-ui-classic/app/views/shared/summary/_textual.html.haml:9:in `___sers_zita__esktop__anage___manageiq_ui_classic_app_views_shared_summary__textual_html_haml__1784846946020889247_70132237067240'
/usr/local/lib/ruby/gems/2.3.0/gems/actionview-5.0.6/lib/action_view/template.rb:159:in `block in render'
/usr/local/lib/ruby/gems/2.3.0/gems/activesupport-5.0.6/lib/active_support/notifications.rb:166:in `instrument'
/usr/local/lib/ruby/gems/2.3.0/gems/actionview-5.0.6/lib/action_view/template.rb:354:in `instrument'
/usr/local/lib/ruby/gems/2.3.0/gems/actionview-5.0.6/lib/action_view/template.rb:157:in `render'
/usr/local/lib/ruby/gems/2.3.0/gems/actionview-5.0.6/lib/action_view/renderer/partial_renderer.rb:343:in `render_partial'
/usr/local/lib/ruby/gems/2.3.0/gems/actionview-5.0.6/lib/action_view/renderer/partial_renderer.rb:311:in `block in render'
/usr/local/lib/ruby/gems/2.3.0/gems/actionview-5.0.6/lib/action_view/renderer/abstract_renderer.rb:42:in `block in instrument'
/usr/local/lib/ruby/gems/2.3.0/gems/activesupport-5.0.6/lib/active_support/notifications.rb:164:in `block in instrument'
/usr/local/lib/ruby/gems/2.3.0/gems/activesupport-5.0.6/lib/active_support/notifications/instrumenter.rb:21:in `instrument'
/usr/local/lib/ruby/gems/2.3.0/gems/activesupport-5.0.6/lib/active_support/notifications.rb:164:in `instrument'
```

@himdel suggests adding `.join(";")` or something :)

Reply:
Fixed, thanks!